homeassistant/components/todo/services.yaml

```diff
@@ -42,5 +42,6 @@ remove_item:
         - todo.TodoListEntityFeature.DELETE_TODO_ITEM
   fields:
     item:
+      required: true
       selector:
         object:
```

Comment:
Should we keep the object selector?

Reply:
I would not, replace it with an text selector for now (single input).